src/app/lib/utilities/filterPopularStaleData/index.js

```diff
@@ -4,26 +4,27 @@ import {
   MOST_READ_STALE_DATA,
 } from '#lib/logger.const';
 import { isDataStale } from './isDataStale';
+import isLive from '../isLive';
 
 const logger = nodeLogger(__filename);
 
 const filterPopularStaleData = ({
   data,
-  isAmp,
-  path,
+  isAmp = false,
+  path = undefined,
   service,
   popularType,
 }) => {
   if (!data) {
     return null;
   }
-  // The ARES test endpoint for most read/watched renders fixture data, so the data is stale
-  const isTest = process.env.SIMORGH_APP_ENV === 'test';
+
   const logEvent =
     popularType === 'mostRead' ? MOST_READ_STALE_DATA : MOST_WATCHED_STALE_DATA;
   // Do not show most read if lastRecordUpdated is greater than 60min as this means PopAPI has failed twice
   // in succession. This suggests ATI may be having issues, hence risk of stale data.
-  if (!isTest && isDataStale(data.lastRecordTimeStamp)) {
+
+  if (isLive() && isDataStale(data.lastRecordTimeStamp)) {
```

Comment:
We can possibly revert these changes now, as we are no longer checking for stale data in the new component